contrib/kind-common

```diff
@@ -0,0 +1,369 @@
+if [ "${BASH_SOURCE[0]}" -ef "$0" ]
```

Comment:
this script should still have a .sh extension to indicate it is bash, shouldn't it?

Reply:
We can, but it is a bit misleading as all it contains is a bunch of functions. It is really meant to be sourced instead of executed. Because of that, I opted to leave the `.sh` out of its name, but it is indeed a bash script, and I will rename it to make that clearer.

Reply:
yeah I actually preferred to not have it as ".sh" ? we have usually not added extension to bash files which shouldn't be sourced..right? it might be mis leading to end users?

Reply:
LOL. Okay... Sorry @trozet , Surya wins and I will revert this change.